libs/traits/src/liquidity_pools.rs

```diff
@@ -22,6 +22,11 @@ pub trait LPEncoding: Sized {
 	fn deserialize(input: &[u8]) -> Result<Self, DispatchError>;
 }
 
+pub trait LPMessage: LPEncoding {
+	fn get_message_proof(&self) -> Option<[u8; 32]>;
+	fn to_message_proof(&self) -> Self;
+}
```

Comment:
I think there is nothing wrong with add them directly to the LPEncoding trait, or did you found something against this?

Reply:
It made sense to me to have a separate trait that defines a separate set of behaviours, but I'm also OK with having just one big trait for this.

Reply:
I have no strong opinion, what you think is better!